portfolio/src/main/webapp/CommentSection.js

```diff
@@ -0,0 +1,29 @@
+ async function initComments()
```

Comment:
This function don't need to be `async`, as you are not using `await` inside.

Reply:
So you haven't deleted `async` in the end?

Reply:
Sorry, I thought I did. I think I did a rebase and it erased it?